apps/google/src/connectors/elba/third-party-apps.ts

```diff
@@ -1,6 +1,35 @@
 import type { ThirdPartyAppsObject } from '@elba-security/sdk';
 import type { GoogleToken } from '../google/tokens';
 
+/**
+ * openid - openid
+ * https://mail.google.com - https://mail.google.com/
+ * https://mail.google.com/ - https://mail.google.com/
+ * https://mail.google.com/something - https://mail.google.com/something
+ * https://mail.google.com/something/ - https://mail.google.com/something
+ */
+
+const cleanScopes = (scopes: string[]): string[] => {
+  return scopes.map((scope) => {
+    if (!URL.canParse(scope)) {
+      return scope;
+    }
+
+    const url = new URL(scope);
+
+    if (url.pathname === '/') {
+      return url.origin + url.pathname;
+    }
+
+    const currentPathname = url.pathname;
+    const normalizedPathname = currentPathname.endsWith('/')
+      ? currentPathname.slice(0, -1)
+      : currentPathname;
+
+    return url.origin + normalizedPathname;
```

Comment:
Not a fan of using `+` to concatenate string. Find it cleaner this way:
```suggestion
    const url = new URL(scope);
    if (url.pathname !== '/' && url.pathname.endsWith('/')) {
      return `${url.origin}${url.pathname.slice(0, -1)}`;
    }

    return `${url.origin}${url.pathname}`;
```